dev/PowerNotifications/PowerNotificationsPal.h

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+
+#include <poclass.h>
+
+struct CompositeBatteryStatus
+{
+    BATTERY_INFORMATION Information;
+    BATTERY_STATUS Status;
+    ULONG ActiveBatteryCount;
+};
+
+DECLARE_HANDLE(CompositeBatteryStatusRegistration);
+HRESULT GetCompositeBatteryStatus(CompositeBatteryStatus** compositeBatteryStatusOut);
+typedef void (*OnCompositeBatteryStatusChanged)(CompositeBatteryStatus* compositeBatteryStatus);
+HRESULT RegisterCompositeBatteryStatusChangedListener(OnCompositeBatteryStatusChanged listener, CompositeBatteryStatusRegistration* registration);
```

Comment:
Where are these methods defined or implemented?  My guess is this is trying out the closed-open boundary system that @EHO-makai and @BenJKuhn are working on.  We should probably discuss how the headers for those components are generated and transferred.

Reply:
Yes, that is correct but as far as I know @brialmsft is working on it.

Reply:
This is effectively a rough draft of the API surface; this header will ultimately be ingested from the closed source NuGet package. We're working on the open-source and closed-source pieces in parallel, since the open-source piece is independently testable with mocks.
With that said, I'm not sure it's appropriate to complete a pull into main while the underlying closed-source code is stubbed out.